models/user.js

```diff
@@ -11,7 +11,11 @@ module.exports = function(sequelize, DataTypes) {
       type: DataTypes.STRING,
       allowNull: true
     },
-    name: {
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
```

Comment:
at the time of signup, only email and password is critical so name should allow null. Same goes for google-login

Reply:
okay lets put it then